extensions/map-layers-auth/src/ArcGis/ArcGisAccessClient.ts

```diff
@@ -227,9 +239,10 @@ export class ArcGisAccessClient implements MapLayerAccessClient {
 
   /**
   * Test if Oauth2 endpoint is accessible and has an associated appId
+  * @return true/false if validation succeeded, undefined if validation could not be performed (i.e CORS/network error)
```

Comment:
is this weird to anyone else? wondering if you should just return false?

Reply:
Wording is a little unclear. It returns true if validation succeeded, false if validation failed, or undefined if validation could not be completed.

Reply:
The thing is, even though we cannot validate it from the iTwin application, the endpoint will work perfectly fine in the login popup, which open directly the oauth2 endpoint (i.e cross origin).

The validate will at least cover cases where the endpoint url would return status 404, or other common http errors.

Reply:
If `undefined` is treated meant to be treated as `true`, and this is internal only, why dont we return `true` on error ? That would simplify code around too and reduce the confusion.

Reply:
> If `undefined` is treated meant to be treated as `true`, and this is internal only, why dont we return `true` on error ? That would simplify code around too and reduce the confusion.

I prefer to be 'transparent' and leave the 'undefined' decision to the caller.  undefined means : we can't tell for sure if it's valid or not.